Tem-Test/global.R

```diff
@@ -0,0 +1,7 @@
+# In global.R we can put things to make sure they are accessible to both ui.R and server.R.
+# In this case, it's data_list and data_choices..
+
+library(dplyr)
+
```

Comment:
Basically moved `data_list` and `data_selected` here from `ui.R`. Otherwise, you will get errors saying `data_list not found` or `data_selected not found`. The only reason we weren't seeing these errors in class is that we were running these apps from a non-empty environment where we already manually created those objects.